keyvi/include/keyvi/util/os_utils.h

```diff
@@ -64,6 +68,15 @@ class OsUtils {
     return limit.rlim_cur;
 #endif
   }
+
+  static inline std::ofstream OpenFileStream(const std::string& filename) {
```

Comment:
Maybe it's worth to add _out_ as part of function name to highlight it is opening an out file stream, like `OpenOutFileStream` ?